data-reconciliation-app/deploy/bicep/mccf.bicep

```diff
@@ -0,0 +1,35 @@
+@description('Resource Name')
```

Comment:
Are we going to use this file as a Bicep deployment sample? I am asking because the PR already has the ARM template.

Reply:
We have `make test-mccf, but it assumes mCCF already exists as we have not added az cli into the devcontainer yet as it will add 1Gb so we cant deploy from the devcontainer.